frontend/public/components/utils/volume-kebab.tsx

```diff
@@ -0,0 +1,30 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Kebab, KebabOption } from './index';
+import { K8sKind, K8sResourceKind, K8sResourceKindReference } from '../../module/k8s';
+import { connectToModel } from '../../kinds';
+import { RowVolumeData } from '../volumes-table';
+
+export const VolumeKebab = connectToModel((props: VolumeKebabProps) => {
```

Comment:
This is not really a generic util since it's only used in one table. I would move it out of the utils folder and into volumes-table.tsx.